config/environments/production.rb

```diff
@@ -35,4 +35,6 @@
   config.logstasher.logger_path = "#{Rails.root}/log/logstash_#{Rails.env}.json"
 
   config.mx_checker = MxChecker.new
+  config.smoke_test_email_local_part = ENV['SMOKE_TEST_EMAIL_LOCAL_PART']
+  config.smoke_test_email_domain = ENV['SMOKE_TEST_EMAIL_DOMAIN']
```

Comment:
It will.  My reasoning here was that both prod and preprod have smoke tests run against them regularly and so should probably not be run without the variables set.  On the other hand, we always use the same addresses, so I think it makes as much sense to got ahead and set them.